tests/system-containers/main.yml

```diff
@@ -465,7 +452,7 @@
 
     - name: Install flannel
       command: >
-        {{ atomic_install_cmd }}
+        {{ aivc_atomic_install_cmd }}
```

Comment:
Maybe add the same comment above this statement about where aivc_atomic_install_cmd comes from?  I'm not sure if its really necessary since its mentioned above but might help someone looking at the code.